### PR TITLE
BSA documentation update: Add quickstart, postman integration, insights definitions. Update overview

### PR DESCRIPTION
**Pull request by gandha9394**

### CODE REVIEW

File: content/data/bank-statement-analyser/quickstart.mdx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+Please send these details via email to [bsa@setu.co](mailto:bsa@setu.co)
+
+Setu will create a report for the FIU and share a `Report ID`, which can be used when [generating insights](#step-3-generate-insights)
+
+### Step 2 - AA Gateway integration
+
+#### Create FI-Datablocks
+
+Create an FI Datablock in Setu's Insights module by making a POST call to `/v1/fi_data` with the data fetched from the FIP
```
Comment (gandha9394) [inline content/data/bank-statement-analyser/quickstart.mdx:33]: fetched from the AA

File: content/data/bank-statement-analyser/quickstart.mdx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+-   The payload should be the same data fetched from the FIP. The schema of this data is validated against the [ReBIT schema](https://api.rebit.org.in/schema)
+-   The API returns a `Data ID` for the datablock along with some metadata, which can be used when [generating insights](#step-3-generate-insights)
+-   A list of available data blocks can also be obtained by making a GET call to `/v1/fi_data`
+
+<Callout type="tip">
+    FI Datablocks will be auto-fetched from the FIP in future.
```
Comment (gandha9394) [inline content/data/bank-statement-analyser/quickstart.mdx:40]: let's remove this for now

File: content/data/bank-statement-analyser/overview.mdx [hunk 3]
```diff
@@ -26,10 +26,21 @@ The analyser modules use statements uploaded via the Account Aggregator flow. Th
 
 4. 50+ indicators for all round financial health: Income analysis, expenditure analysis and categorization engines that allow to create a complete picture of an individual’s financial health
 
+### How does it work?
+
+![](https://storage.googleapis.com/strapi-assets/latest/setu-bsa-how-it-works-1/setu-bsa-how-it-works-1.png)
+
+First, customer gives consent to the FIU to access bank details from the FIP. The FIP then sends the bank statement data to the FIU via the AA gateway. But before this raw data goes to the FIU, it is routed via the Setu Bank Statement Analyser layer.
```
[the remark this answers is not in the document]
Comment (shashi-budati) [reply content/data/bank-statement-analyser/overview.mdx]: Added link to AA's glossary

File: content/data/bank-statement-analyser/overview.mdx [hunk 4]
```diff
@@ -26,10 +26,21 @@ The analyser modules use statements uploaded via the Account Aggregator flow. Th
...
+
+Here, the raw data goes through a categorization model built by Setu, and gets bucketed into various categories of incomes, expenses etc.
```
[the remark this answers is not in the document]
Comment (shashi-budati) [reply content/data/bank-statement-analyser/overview.mdx]: Updated with headings

File: content/data/bank-statement-analyser/quickstart.mdx [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+Here is a quick summary of steps to help you integrate with Setu’s Insights API. Before you start, there are some key terms that will be referred to periodically in the documentation—
+
+-   **FI Datablock**— FI Datablocks is the data received from the FIP (Financial Information Provider), via the AA. Currently, we only support bank statement data.
+-   **Report**— A report is the template in which the FIU wants the insights to be generated. An FIU can pick and choose exactly what modules should be added to a report based on their needs. For example, if an FIU only wants to get the income and expenses of a customer, the report can contain only those two modules.
+
+<hr className="primary" />
+
+### Step 1 - Report creation
+
+FIU needs to provide Setu with the following items for configuration
```
[the remark this answers is not in the document]
Comment (shashi-budati) [reply content/data/bank-statement-analyser/quickstart.mdx]: Updated

File: content/data/bank-statement-analyser/quickstart.mdx [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+
+-   **FIU ID** received from the integration with AA
+-   **Name** for the report, and a list of required <a href="/data/bank-statement-analyser/postman" target="_blank">**Insights**</a>
+-   **Format** of the report's output (currently defaults to `JSON`)
+
+Please send these details via email to [bsa@setu.co](mailto:bsa@setu.co)
+
+Setu will create a report for the FIU and share a `Report ID`, which can be used when [generating insights](#step-3-generate-insights)
```
[the remark this answers is not in the document]
Comment (shashi-budati) [reply content/data/bank-statement-analyser/quickstart.mdx]: Updated

**Pull request merged**